roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -100,6 +100,22 @@
   when:
     - '"lbaas" in enable_services'
 
+- name: create devstack local conf with octavia enabled
```

Comment:
We have added the octavia in above section(L86~L101), which add both neutron-lbaas and octavia,  do you think we need a separated `octavia` support ?

Reply:
I'm ok to add new 'octavia' service and keep 'lbaas' service still be in here, because OpenLab is a platform, should support different scenairo to different SDK/Tool, even if OpenStack community have deprecated neturon-lbaas in latest release, but still have some SDK/Tools use old release and build logic on neutron-lbaas API, like: cloud-provider-openstack. https://github.com/kubernetes/cloud-provider-openstack/blob/2bbd88dd806e63b2311898dc01b2cafe279efc2e/docs/cloud-providers.md#load-balancer

Reply:
Yes, Chen Rui made the point.